### ISSUE TITLE
16544594: VoiceOver scrolls off screen when selecting a hidden element on a page

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#3737]
#### Description

Summary:
VoiceOver scrolls off screen when selecting a hidden element on a page. This appears to be a new issue in iOS 7, it did not occur in iOS 6.

Steps to Reproduce:
1. Start VoiceOver
2. Open Safari
3. Navigate to www.google.com
4. Search for 'something'
5. Tap the top most result on the page [Something (Song - the Beatles)]
6. Swipe left to navigate backwards
7. VoiceOver focus moves to a hidden heading element and the rest of the page disappears. Tapping on the blank space results in nothing being read out (because there's nothing there).
8. Swipe left again and focus moves to the previous visible text on the page ('More'). If instead of continuing to swipe left/right the user taps outside the VoiceOver cursor and then tries to navigate by swiping lose the page and cannot get back to it. Focus switches to the Safari header or footer and the page is not accessible without reloading.

Expected Results:
Expect VoiceOver focus not to move offscreen to a hidden element. Hidden element should be read out without having to move focus off screen.

Actual Results:
VoiceOver moves focus off screen.
## 

Product Version: 7
Created: 2014-04-07T21:48:14.548648
Originated: 2014-04-07T00:00:00
Open Radar Link: http://www.openradar.me/16544594
